sdk/storage/src/table/clients/partition_key_client.rs

```diff
@@ -67,6 +67,8 @@ impl PartitionKeyClient {
 
 #[cfg(test)]
 #[cfg(feature = "test_integration")]
+// TODO #297 enable test once transactions are working in Azurite
+#[cfg(feature = "azurite_todo")]
```

Comment:
Are we doing this todo as a feature so we can easily re-enable for testing? I guess I would assume we'd just use `#[ignore = "enable when azure is working with transactions"]`

Reply:
Using `ignore` makes more sense to me. I updated the PR. FYI, I did have to add `unused_attributes` to `RUSTFLAGS: --deny warnings --allow unused_attributes`.